sound/soc/sof/intel/hda.c

```diff
@@ -1305,6 +1305,7 @@ struct snd_soc_acpi_mach *hda_machine_select(struct snd_sof_dev *sdev)
 	u32 interface_mask = hda_get_interface_mask(sdev);
 	struct snd_sof_pdata *sof_pdata = sdev->pdata;
 	const struct sof_dev_desc *desc = sof_pdata->desc;
+	struct hdac_bus *bus = sof_to_bus(sdev);
```

Comment:
Looks good @plbossart . Just a small typo in the commit message, correc,t

Reply:
updated, thanks for noticing this @ranj063